.github/workflows/main.yaml

```diff
@@ -0,0 +1,48 @@
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Нужно добавить `cache`, чтобы `sbt` из `cache` можно было использовать
Сейчас так ([по логам CI](https://github.com/mipt-backend-academy-winter-2023/team-3/actions/runs/6978163390/job/18989095415?pr=19#step:3:7)):
```
[info] [launcher] getting org.scala-sbt sbt 1.9.6  (this may take some time)...
```

Reply:
вроде поправил